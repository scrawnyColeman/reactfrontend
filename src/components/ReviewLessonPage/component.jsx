import React from 'react';
import styled from 'styled-components';

const StyledContainer = styled.div`
    margin-top: 20vh;
`;

const ReviewLessonPage = () => <StyledContainer>memous supremous</StyledContainer>;

export default ReviewLessonPage;
