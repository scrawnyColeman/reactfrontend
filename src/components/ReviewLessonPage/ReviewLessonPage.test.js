import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-styled-components';
import React from 'react';
import ReviewLessonPage from './component';

Enzyme.configure({ adapter: new Adapter() });

describe('ReviewLessonPage component', () => {
    test('component matches snapshot', () => {
        const wrapper = shallow(<ReviewLessonPage />);
        expect(wrapper).toMatchSnapshot();
    });
});
