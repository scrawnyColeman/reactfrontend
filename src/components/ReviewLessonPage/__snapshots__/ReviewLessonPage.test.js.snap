// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewLessonPage component component matches snapshot 1`] = `
<styled.div>
  memous supremous
</styled.div>
`;
